cli/commands/completeAllWithdrawals.go

```diff
@@ -0,0 +1,142 @@
+package commands
+
+import (
+	"context"
+	"fmt"
+	"math/big"
+
+	lo "github.com/samber/lo"
+
+	"github.com/Layr-Labs/eigenlayer-contracts/pkg/bindings/EigenPod"
+	"github.com/Layr-Labs/eigenlayer-contracts/pkg/bindings/IDelegationManager"
+	"github.com/Layr-Labs/eigenpod-proofs-generation/cli/core"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+)
+
+type TCompleteWithdrawalArgs struct {
+	EthNode     string
+	EigenPod    string
+	Sender      string
+	EstimateGas bool
+}
+
+func DelegationManager(chainId *big.Int) common.Address {
+	data := map[uint64]string{
+		// TODO(zeus) - make this runnable via zeus.
+		1:     "0x39053D51B77DC0d36036Fc1fCc8Cb819df8Ef37A", // mainnet
+		17000: "0x75dfE5B44C2E530568001400D3f704bC8AE350CC", // holesky preprod
```

Comment:
this will break for testnet

Reply:
yup -- this also won't work on testnet currently.

Planning on swapping this address out once we deploy testnet.

Reply:
Generally the CLI only supports testnet + mainnet.